ui/packages/ui/src/app/pages/createConnector/CreateConnectorComponent.tsx

```diff
@@ -23,7 +23,7 @@ import {
 import _ from "lodash";
 import React, { Dispatch, ReactNode, SetStateAction } from "react";
 import { useTranslation } from "react-i18next";
-import { useHistory } from "react-router-dom";
+import { Prompt, useHistory } from "react-router-dom";
```

Comment:
useHistory can be removed - not used

Reply:
yeah, I missed them. I will do the clean up in the subsequent commit.

Reply:
done